src/code/qualification/getConfigurationPlan.m

```diff
@@ -14,10 +14,10 @@
 % Read .json file and output its structure
 
 try
-    currentEncoding = slCharacterEncoding;
-    slCharacterEncoding('UTF-8');
+    currentEncoding = feature('DefaultCharacterSet');
+    feature('DefaultCharacterSet', 'UTF8');

```

Comment:
another (undocumented) way to change Matlab character encoding.
Changed because `slCharacterEncoding `is not supported by the Matlab Compiler